spec/features/admin/merchants/index_spec.rb

```diff
@@ -63,4 +63,8 @@
       expect(page).to_not have_content(merchant_3.name)
     end
   end
+
+  it 'has a link to create a new merchant on a new page' do
+    expect(page).to have_link("Create a New Merchant", href: "/admin/merchants/new")
```

Comment:
I haven't seen a test for a link written like this before.  Interesting!

Reply:
Oooh me neither! Good idea!